pkg/sdk/validation/manifest/fsvalidator_test.go

```diff
@@ -44,32 +42,36 @@ func TestFilesystemValidator_ValidateFile(t *testing.T) {
 		"Valid Interface": {
 			manifestPath:                "testdata/valid-interface.yaml",
 			expectedValidationErrorMsgs: []string{},
-			hubCli: fixHub(t, map[graphql.ManifestReference]bool{
+			hubCli: fixHubForManifestsExistence(t, map[graphql.ManifestReference]bool{
 				manifestRef("cap.type.productivity.mattermost.config"):        true,
 				manifestRef("cap.type.productivity.mattermost.install-input"): true,
 			}, nil),
 		},
 		"Valid Type": {
 			manifestPath:                "testdata/valid-type.yaml",
 			expectedValidationErrorMsgs: []string{},
-			hubCli:                      fixHub(t, map[graphql.ManifestReference]bool{sampleAttr: true}, nil),
+			hubCli:                      fixHubForManifestsExistence(t, map[graphql.ManifestReference]bool{sampleAttr: true}, nil),
 		},
 		"Invalid Implementation": {
 			manifestPath: "testdata/invalid-implementation.yaml",
 			expectedValidationErrorMsgs: []string{
 				"OCFSchemaValidator: spec: appVersion is required",
+				"RemoteImplementationValidator: Type cap.type.platform.cloud-foundry:0.1.0 is not attached to cap.core.type.platform abstract node",
 				"RemoteImplementationValidator: manifest revision 'cap.interface.cms.wordpress:0.1.0' doesn't exist in Hub",
 			},
-			hubCli: fixHub(t, map[graphql.ManifestReference]bool{
-				manifestRef("cap.interface.cms.wordpress"): false,
+			hubCli: fixHub(t, []*graphql.Type{
+				fixGQLType("cap.type.platform.cloud-foundry", "0.1.0", ""),
+			}, map[graphql.ManifestReference]bool{
+				manifestRef("cap.interface.cms.wordpress"):     false,
+				manifestRef("cap.type.platform.cloud-foundry"): true,
 			}, nil),
 		},
```

Comment:
It might be quite confusing example, as logically cloud-foundry should be (and is) attached in real life. Maybe rename that to different requirement in the `invalid-implementation.yaml`, e.g.:

```
  requires:
    cap.core.type.platform:
      oneOf:
        - name: cap.type.database.postgresql.config
          revision: 0.1.0
```